src/macaron/slsa_analyzer/git_url.py

```diff
@@ -327,25 +327,11 @@ def clone_remote_repo(clone_dir: str, url: str) -> Repo | None:
     # Macaron attempting to clone a repository multiple times.
     # In these cases, we should not error since it may interrupt the analysis.
     if os.path.isdir(clone_dir):
-        logger.info("Checking if the repo %s needs cloning.", url)
         try:
             os.rmdir(clone_dir)
-            logger.info(
-                "The clone dir %s is empty. It has been deleted for cloning the repo %s.",
-                clone_dir,
-                url,
-            )
         except OSError:
-            logger.info(
```

Comment:
Same comment as above, and log at debug level.

Reply:
Reverted in https://github.com/oracle/macaron/commit/324de16baf5b72492a91c353a9c67fd4a92cbeac